Formula/memcached.rb

```diff
@@ -25,36 +25,25 @@ def install
     system "make", "install"
   end
 
-  plist_options manual: "#{HOMEBREW_PREFIX}/opt/memcached/bin/memcached"
-
-  def plist
-    <<~EOS
-      <?xml version="1.0" encoding="UTF-8"?>
-      <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
-      <plist version="1.0">
-      <dict>
-        <key>Label</key>
-        <string>#{plist_name}</string>
-        <key>KeepAlive</key>
-        <true/>
-        <key>ProgramArguments</key>
-        <array>
-          <string>#{opt_bin}/memcached</string>
-          <string>-l</string>
-          <string>localhost</string>
-        </array>
-        <key>RunAtLoad</key>
-        <true/>
-        <key>WorkingDirectory</key>
-        <string>#{HOMEBREW_PREFIX}</string>
-      </dict>
-      </plist>
-    EOS
+  service do
+    run [opt_bin/"memcached"]
```

Comment:
What happened to the `-l localhost` flags? I think we also still need to set `working_dir`. Not sure about `keep_alive` and `run_type`, maybe `true` and `:immediate` are the defaults there.

Reply:
Whoops, fixed all that.

Reply:
keep_alive defaults to `false` iirc and `run_type` isn't needed for now (until I start to implement timers)